site/src/nav/DarkModeToggle.jsx

```diff
@@ -14,7 +14,7 @@ export default function DarkModeToggle({ mode, setMode }) {
   };
 
   return (
-    <div className="dark-mode-toggle">
+    <div className="dark-mode-toggle tour-darkmode">
```

Comment:
Added as tour target

Reply:
One potential issue- what if we're in light mode? Appreciate the attention to making sure we are mode-aware!

Reply:
Screenshot of tour in light mode: 
![image](https://github.com/OvertureMaps/io-site/assets/1887955/27e37906-53e1-46c7-bbd7-a7a3be5b0184)

Reply:
This should be explicitly fixed now. Before, the coloring was dependent on the browser color scheme, but now it is dependent on the site color scheme.